proxy/src/serve_tasks.rs

```diff
@@ -173,7 +301,7 @@ async fn sign_request(
     parts.uri = target_uri.clone();
     headers_mut.insert(header::HOST, config.broker_host_header.clone());
 
-    let length = body.size_hint().exact().ok_or_else(|| {error!("Cannot calculate length of request"); ERR_BODY})?;
+    let length = HttpBody::size_hint(&body).exact().ok_or_else(|| {error!("Cannot calculate length of request"); ERR_BODY})?;
```

Comment:
Is this a functional change or just to improve code readibility?

Reply:
This is needed to disambiguate two implementations of the same name `size_hint()` coming from two traits.